certora/ArrangerConduit.conf

```diff
@@ -0,0 +1,28 @@
+{
+    "files": [
+        "src/ArrangerConduit.sol",
+        "lib/dss-allocator/src/AllocatorRoles.sol",
+        "lib/dss-allocator/src/AllocatorRegistry.sol",
+        "lib/mock-erc20/src/MockERC20.sol",
+        "certora/Auxiliar.sol"
+
+    ],
+    "link": [
+        "ArrangerConduit:roles=AllocatorRoles",
+        "ArrangerConduit:registry=AllocatorRegistry",
+    ],
+    "optimistic_loop": true,
+    "rule_sanity": "basic",
+    "solc_optimize_map": {
```

Comment:
It's missing to specify a general solc version or a map of them.

Reply:
Done